atomate/vasp/fireworks/core.py

```diff
@@ -130,6 +134,42 @@ def __init__(self, structure, name="static", vasp_input_set=None,
             structure.composition.reduced_formula, name), **kwargs)
 
 
+class StaticInterpolateFW(Firework):
+    def __init__(self, structure, start, end, name="static", vasp_input_set="MPStaticSet",
+                 vasp_input_set_params=None, vasp_cmd="vasp", db_file=None,
+                 parents=None, this_image=None, nimages=None, autosort_tol=0, **kwargs):
+        """
+        Standard static calculation Firework that interpolates structures from two previous calculations.
+
+        Args:
+            structure (Structure): Input structure used to name FireWork.
+            name (str): Name for the Firework.
```

Comment:
document ``start`` and ``end`` params

Reply:
also just make sure the docs and parameters are consistent overall here

Reply:
Done.